node-launchpad/src/config.rs

```diff
@@ -17,6 +18,26 @@ use std::path::PathBuf;
 
 const CONFIG: &str = include_str!("../.config/config.json5");
 
+// Where to store the Nodes data
+pub fn get_launchpad_nodes_data_dir_path(base_dir: PathBuf) -> Result<PathBuf> {
+    // We get the data dir and we remove the first character which is the / character
+    let mut data_directory = dirs_next::data_dir()
+        .expect("Data directory is obtainable")
+        .to_str()
+        .unwrap()
+        .to_string();
+    data_directory.remove(0);
```

Comment:
`data_dir()` would include the `$HOME` folder, I don't think we should recreate the same strucutre on a different drive. It might be confusing for people and I have not seen that anywhere. I.e., we'd get `F:\Users\Alice\AppData\Roaming`. 

Can we just use `F:\safe\node`?

Reply:
Yep, we can use just `F:\safe\node`. My only concern is how do we know that is not the default drive the one that we are using so we don't use the `$HOME` path. We can discuss.

Reply:
We've agreed that `/` or `C:/` is the default mount point, and they'd use the `data_dir()` structure, whereas the other mounts would use `$MOUNT/safe/node` structure.